### PR TITLE
Fix base.html.twig path

### PR DESCRIPTION
**Pull request by quentint**
To match Symfony 3.4 structure, as seen here: https://symfony.com/doc/3.4/templating.html

### CODE REVIEW

File: frontend/encore/simple-example.rst [hunk 1]
```diff
@@ -82,7 +82,7 @@ your layout. In Symfony, use the ``asset()`` helper:
 
 .. code-block:: twig
 
-    {# templates/base.html.twig #}
+    {# app/Resources/views/base.html.twig #}
```
Comment (yceruto) [inline frontend/encore/simple-example.rst]: both are valid for `3.4` though.

Comment (quentint) [reply frontend/encore/simple-example.rst]: `symfony new my_project 3.4` comes with `app/Resources/views/base.html.twig`, and the docs reference `app/Resources/views/` a lot.

It makes more sense to me. Don't you think?

Comment (yceruto) [reply frontend/encore/simple-example.rst]: Sure! for consistency :+1:

**Comment by javiereguiluz** [comment]
@quentint you are right! In 3.4 branch we still show the old template path. Thanks for fixing this ... and congrats on your first Symfony Docs contribution!

**Pull request merged**